Formula/wordgrinder.rb

```diff
@@ -12,12 +13,15 @@ class Wordgrinder < Formula
     sha256 "19855507ccd289049c30aad466a66de742b9d58a98247b9f9d223e140a81ff69" => :high_sierra
   end
 
+  depends_on "ncurses" => :build
   depends_on "ninja" => :build
   depends_on "lua"
 
   uses_from_macos "zlib"
 
   def install
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["ncurses"].opt_lib/"pkgconfig"
```

Comment:
This line shouldn’t be needed. Homebrew by default adds the pkgconfig folders of dependencies.